src/utils/rest-api.js

```diff
@@ -1237,6 +1285,7 @@ export function createVoltageLevel(
             substationId: substationId,
             busbarSections: busBarSections,
             busbarConnections: busBarConnections,
+            uuid,
```

Comment:
Maybe put this uuid in the URL as the pathVariable when it's a PUT request.
And don't add it if it's a POST because it's always null in this case.